src/active_item_cache.cpp

```diff
@@ -48,7 +48,12 @@ void active_item_cache::add( item &it, point location )
 
 bool active_item_cache::empty() const
 {
-    return active_items.empty();
+    for( std::pair<int, std::list<item_reference>> active_queue : active_items ) {
```

Comment:
Now that I see the reverting PR, this should probably iterate over a reference, not a copy.